manim/mobject/svg/svg_mobject.py

```diff
@@ -28,6 +29,97 @@ def string_to_numbers(num_string):
     return [float(s) for s in re.split("[ ,]", num_string) if s != ""]
 
 
+def cascade_element_style(element, inherited):
+    """Collect the element's style attributes based upon both its inheritance and its own attributes.
+
+    SVG uses cascading element styles. A closer ancestor's style takes precedence over a more distant ancestor's
+    style. In order to correctly calculate the styles, the attributes are passed down through the inheritance tree,
+    updating where necessary.
+
+    Note that this method only copies the values and does not parse them. See :meth:`parse_color_string` for converting
+    from SVG attributes to manim keyword arguments.
+    """
+
+    style = dict(inherited)
+
+    styling_attributes = ["fill", "stroke", "style", "fill-opacity", "stroke-opacity"]
+    for attr in styling_attributes:
+        entry = element.getAttribute(attr)
+        if entry:
+            style[attr] = entry
+
+    return style
+
+
+def parse_color_string(color_spec):
+    """Handle the SVG-specific color strings and convert them to HTML #rrggbb format."""
+
+    if color_spec[0:3] == "rgb":
+        # these are only in integer form, but the Colour module wants them in floats.
+        splits = color_spec[4:-1].split(",")
+        if splits[0][-1] == "%":
+            # if the last character of the first number is a percentage,
+            # then interpret the number as a percentage
+            parsed_rgbs = [int(i[:-1]) / 100.0 for i in splits]
+        else:
+            parsed_rgbs = [int(i) / 255.0 for i in splits]
+
+        hex_color = rgb_to_hex(parsed_rgbs)
+
+    elif color_spec[0] == "#":
+        # its OK, parse as hex color standard.
+        hex_color = color_spec
+
+    else:
+        # attempt to convert color names like "red" to hex color
+        hex_color = web2hex(color_spec, force_long=True)
+
+    return hex_color
+
+
+def parse_style(svg_style):
+    """Convert a dictionary of SVG attributes to Manim VMobject keyword arguments."""
+
+    manim_style = dict()
```

Comment:
I would like to do so
```suggestion
    manim_style = {}
```